packages/@aws-cdk/aws-lambda-go/test/lambda-handler-vendor/go.mod

```diff
@@ -4,7 +4,13 @@ go 1.16
 
 require (
 	github.com/aws/aws-lambda-go v1.19.1
+	github.com/go-check/check v0.0.0-20200902074654-038fdea0a05b // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
-	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
+)
+
+replace (
+	gopkg.in/check.v1 => github.com/go-check/check v0.0.0-20200902074654-038fdea0a05b
+	gopkg.in/yaml.v2 => github.com/go-yaml/yaml v2.1.0+incompatible
```

Comment:
Looking at the [`2.1.0`](https://github.com/go-yaml/yaml/tree/v2.1.0) tag of this repo, it looks like it doesn't contain a `go.mod` file, which is why we get the `+incompatible` mark I believe. 

Lets switch to [`2.1.4`](https://github.com/go-yaml/yaml/tree/v2.4.0), which looks ok from that perspective.